internal/ddt/node.go

```diff
@@ -372,21 +394,10 @@ func (n *Node) Assets() ([]*NodeAsset, error) {
 	}
 
 	for _, f := range files {
-		if f.IsDir() {
-			continue
-		}
-		if strings.HasPrefix(f.Name(), ".") {
-			continue
-		}
-		if NodeMetaRegexp.MatchString(f.Name()) {
-			continue
-		}
-		if NodeDocsRegexp.MatchString(f.Name()) {
-			continue
-		}
-		if NodeAssetsIgnoreRegexp.MatchString(f.Name()) {
```

Comment:
See my comment above, in which we use `Assets()` directly and let it do the pre-filtering.